ground/gcs/src/plugins/uavobjectbrowser/treeitem.cpp

```diff
@@ -47,7 +47,8 @@ HighLightManager::HighLightManager(long checkingInterval, QTime *currentTime)
 
     if (currentTime == NULL)
         m_currentTime = new QTime;
-    m_currentTime = currentTime;
+
+    *m_currentTime = *currentTime;
```

Comment:
This stores the value, as opposed to immediately overwriting the allocated pointer with something we don't own.